src/card/basic/card.html

```diff
@@ -1,42 +1,33 @@
 <div ng-class="showTopBorder === 'true' ? 'card-pf card-pf-accented' : 'card-pf'">
-  <div ng-if="showHeader()"
-       ng-class="!showTitlesSeparator || showTitlesSeparator === 'true' ? 'card-pf-heading' : 'card-pf-heading-no-bottom'">
-    <div class="card-pf-table">
-      <div class="card-pf-cell">
-        <h2 class="card-pf-title" ng-class="{'card-pf-filter-header': showFilterInHeader()}">{{headTitle}}</h2>
-      </div>
-      <div class="card-pf-cell text-right" ng-if="showFilterInHeader()">
-        <div class="dropdown btn-group">
-          <div ng-include="'card/basic/card-filter.html'"></div>
-        </div>
-      </div>
+  <div ng-if="showHeader()" ng-class="{'card-pf-heading': shouldShowTitlesSeparator, 'card-pf-heading-no-bottom': !shouldShowTitlesSeparator}">
+    <div ng-if="showFilterInHeader()" class="dropdown card-pf-time-frame-filter">
```

Comment:
This might be easier to read
ng-class="shouldShowTilesSeparator ? 'card-pf-heading' | 'card-pf-heading-no-bottom'"